sklearn/compose/_column_transformer.py

```diff
@@ -343,14 +343,20 @@ def get_feature_names(self):
         """
         check_is_fitted(self, 'transformers_')
         feature_names = []
-        for name, trans, _, _ in self._iter(fitted=True):
+        for name, trans, column, _ in self._iter(fitted=True):
             if trans == 'drop':
                 continue
-            elif trans == 'passthrough':
-                raise NotImplementedError(
-                    "get_feature_names is not yet supported when using "
-                    "a 'passthrough' transformer.")
-            elif not hasattr(trans, 'get_feature_names'):
+            if trans == 'passthrough':
+                if isinstance(column, (str, int)):
+                    column = [column]
```

Comment:
I think test coverage dropped because this has not been tested. This can be tested once #14495 is added, or these 2 lines can be removed if it is not added.

Reply:
Fixed coverage issue, as #14495 was closed